src/SymbolPaths/include/SymbolPaths/PersistentStorageManager.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOL_PATHS_PERSISTENT_STORAGE_MANAGER_H
+#define SYMBOL_PATHS_PERSISTENT_STORAGE_MANAGER_H
+
+#include <filesystem>
+#include <memory>
+#include <vector>
+
+namespace orbit_symbol_paths {
+
+class PersistentStorageManager {
+ public:
+  PersistentStorageManager() = default;
+  virtual ~PersistentStorageManager() = default;
+
+  // This is a simple wrapper around QSettings, which makes this thread safe, see
+  // https://doc.qt.io/qt-5/qsettings.html
```

Comment:
Since this defines an interface the comment doesn't make a lot of sense. The whole point of this interface to have it not depend on QSettings

Reply:
done in #3347